ops/staging-deploy.tmpl.yaml

```diff
@@ -79,14 +83,22 @@ extraEnvVars: &envVars
     value: $DATABASE_PASSWORD
   - name: DATABASE_USER
     value: hydra_user
+  - name: DB_HOST
+    value: hyrax20181216.cn4nhxyoxlnu.us-west-2.rds.amazonaws.com
+  - name: DB_PORT
+    value: '5432'
+  - name: DB_USERNAME
+    value: hydra_user
   - name: FCREPO_BASE_PATH
     value: /staging
   - name: FCREPO_HOST
-    value: fcrepo.default
+    value: fcrepo.fcrepo.svc.cluster.local
```

Comment:
I see lots off `fcrepo.fcrepo`, `solr.solr`, etc so I assume this is intentional. Why are there two though?

Reply:
thank you it should be the same throughout, but the default does eventually make it!
the second instance of `fcrepo` in this: fcrepo.fcrepo.svc.cluster.local, typically refers tot he namespace the workload is deployed to in the cluster.